Formula/braid.rb

```diff
@@ -0,0 +1,68 @@
+class Braid < Formula
+  desc "Simple tool to help track vendor branches in a Git repository"
+  homepage "https://cristibalan.github.io/braid/"
+  url "https://rubygems.org/gems/braid-1.0.21.gem"
```

Comment:
I'd prefer we use `https://github.com/cristibalan/braid/archive/v1.0.21.tar.gz`

Reply:
Great. Sounds good to me